scripts/setup.sh

```diff
@@ -33,23 +33,23 @@ MODULES=(
 
 # functions
 INSTALL_MODULE() {
-    $SYSREPOCTL -a -i $MODDIR/$1 -s $MODDIR -p $PERMS -o $OWNER -g $GROUP -v2
+    "$SYSREPOCTL" -a -i $MODDIR/$1 -s "$MODDIR" -p "$PERMS" -o "$OWNER" -g "$GROUP" -v2
```

Comment:
I did this initially, but `$1` also contains feature enabling arguments via `-e` so here's how an install command looks if I do this:

/opt/sysrepo/bin/sysrepoctl -a -i '/opt/Netopeer2/share/yang/modules/netopeer2/ietf-netconf-server@2019-07-02.yang -e ssh-listen -e tls-listen -e ssh-call-home -e tls-call-home' -s /opt/Netopeer2/share/yang/modules/netopeer2 -p 600 -o andrei.pavel -g 'domain users' -v2

It fails saying it can't find module `<module> -e ssh-listen -e tls-listen -e ssh-call-home -e tls-call-home` which is the expansion of `$MODDIR/$1`.

I could separate module path from feature enables, but let me know if you want this and then let's do this in a separate pull request?

Reply:
Right, missed that. I left it together because I did not really had time to play around with it to make the features separate but at the same time make their connection to a specific module obvious. If you can do that, be my guest, but I think it is okay the way it is now.